service/src/directory.js

```diff
@@ -36,7 +36,8 @@ exports.getUsers = async (req) => {
         objectType: "user" ,
         page: page,
       });
-      users = users.concat(response.results);
+      const jsonResponse = toJson(GetObjectsResponseSchema, response);
```

Comment:
Do we need to do this explicitly, or could we just build an array by concatenating each page results?

Reply:
The issue with the array is that it won't serialize correctly toJson, an alternative would be to create an empty `GetObjectsResponse` and concatenate the users in the results array
```ts
const usersResponse = create(GetObjectsResponseSchema, { results: [] });
usersResponse.push(...users.results)
```

Reply:
The thing I'm trying to avoid (if possible) is to have the user require knowledge about `GetObjectResponseSchema`, which for a javascript program is confusing (why do I need to know about a schema?)

I understand the challenge - a "naked" array doesn't know how to serialize the extra garbage that the connect library attaches to the objects in that array (hence the BigInt serialization error). 

I wonder if the `response.results` array that we return should have only JSON objects. This completely hides the implementation details from the user.

Is there a reason why they would need to know about the rest of the connect junk?
![Screenshot 2024-12-17 at 10 59 22 AM](https://github.com/user-attachments/assets/49e6175d-3f05-41b9-9e67-07cef0e23c52)